packages/components/top-menu/top-menu.ts

```diff
@@ -0,0 +1,59 @@
+import {
+    booleanAttribute,
+    ChangeDetectionStrategy,
+    Component,
+    Directive,
+    input,
+    Input,
+    ViewEncapsulation
+} from '@angular/core';
+
+@Directive({
+    standalone: true,
+    selector: '[kbq-top-menu-spacer]',
```

Comment:
```suggestion
    selector: '[kbqTopMenuSpacer]',
```
[styleguide](https://angular.dev/style-guide#style-02-08)